### PR TITLE
Add the AmazonSSMManagedInstanceCore policy to the instance role for the SSM agent

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request adds the AWS-provided `AmazonSSMManagedInstanceCore` policy to the OpenVPN server instance role for the SSM agent.

## 💭 Motivation and Context

With [aws/amazon-ssm-agent](https://github.com/aws/amazon-ssm-agent) we can get a shell on any instance (with or without `ssh`) via the AWS control plane.  This means we do not need to open port 22 or even provide a public IP or Internet Gateway.

## 🧪 Testing

All the pre-commit hooks pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/lineage.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```
Comment (dav3r) [inline .github/lineage.yml]: This should be a descendant of `skeleton-tf-module`.

**Review by dav3r — APPROVED** [review]
Approved once you fix the skeleton fu. 🦴

**Review by felddy — APPROVED** [review]
☮️

File: .github/lineage.yml (hunk 1 above)
Comment (jsf9k) [reply .github/lineage.yml]: See commit 8dddb16.

**Review by hillaryj — APPROVED** [review]
🎉

**Pull request merged**